CONTRIBUTING.md

```diff
@@ -5,9 +5,14 @@ email, or any other method with the owners of this repository before making a ch
 
 Please note we have a code of conduct, please follow it in all your interactions with the project.
```

Comment:
Welcome to our community. Please follow our Code of Conduct in all of your interactions with the project.